requirements/constraints.txt

```diff
@@ -10,3 +10,6 @@
 
  # These packages are backports which can only be installed on Python 2.7
 futures ; python_version == "2.7"
+Django<3.0
+mock<4.0  # requires python>3.5
+zipp<2.0
```

Comment:
Please add a comment for all constraints to explain why they are needed, which will make it simpler to know when they can go away.  Thanks.